.github/workflows/osx_testing.yml

```diff
@@ -53,20 +42,16 @@ jobs:
         run: brew install tarantool
         if: matrix.tarantool == 'brew'
 
+      - name: Setup python3 for tests
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+
       - name: Cache built tarantool ${{ env.T_VERSION }}
-        uses: actions/cache@v2
+        uses: actions/cache@v4
         id: cache
         with:
           path: ${{ env.T_DESTDIR }}
-          # v2 is due to https://github.com/actions/cache/issues/2
-          # and because the cache keys without -v2 may contain
-          # debug tarantool builds. It is desirable to have all
-          # build either debug or release (RelWithDebInfo), but
-          # we unable to build all releases in debug (see below).
-          #
-          # v3 is to re-verify all Mac OS builds after fix for the
-          # gh-6076 problem (see below).
-          #
```

Comment:
It is about the `key` version below, not `actions/cache@v<...>`.

It is not so necessary after https://github.blog/changelog/2022-10-20-manage-caches-in-your-actions-workflows-from-web-interface/ , so you can drop this versioning if you want.

Reply:
Dropped it and updates the commit message:
```diff
diff --git a/.github/workflows/osx_testing.yml b/.github/workflows/osx_testing.yml
index 6144357..dc6c1e6 100644
--- a/.github/workflows/osx_testing.yml
+++ b/.github/workflows/osx_testing.yml
@@ -52,9 +52,7 @@ jobs:
         id: cache
         with:
           path: ${{ env.T_DESTDIR }}
-          # v4 added due to inability to clear the cache after v3 prefix.
-          # See https://github.com/github/docs/issues/14145
-          key: ${{ matrix.runs-on }}-${{ matrix.tarantool }}-v4
+          key: ${{ matrix.runs-on }}-${{ matrix.tarantool }}
         if: matrix.tarantool != 'brew' && matrix.tarantool != 'master'
 
       - name: Install tarantool build dependencies
```